README.md

```diff
@@ -1,8 +1,21 @@
 # This is public data
 
 ## installation
+After Successfully setting up the student-intern-organiser Application repository 
+Do the Following to setup the Database
+1. Change Directory/Folder
+```
+cd student-intern-organiser
+```
+2. clone the repository
+```
+git clone https://github.com/WEHI-ResearchComputing/student_intern_data_public.git
+```
+4. Create a Directory/Folder named  **student_intern_data**
+```
+mkdir student_intern_data/
+```
+4. Copy All from **student_intern_data_public** to **student_intern_data**
+
+Enjoy the Run
 
-- git clone git@github.com:WEHI-ResearchComputing/student-intern-organiser.git public_student_intern_organiser
-- cd public_student_intern_organiser/
-- git clone git@github.com:WEHI-ResearchComputing/student_intern_data_public.git student_intern_data
```

Comment:
See this line - this does all that for you :)

Reply:
yes  thats true , i was only changing from using ssh url to https url